Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -664,9 +661,30 @@ def list_users_accounts_command(client: Client, args: dict):
 
 
 def format_fetch_start_time_to_timestamp(fetch_start_time: Optional[str]):
-    first_fetch_dt = parse(fetch_start_time).replace(tzinfo=utc)  # type:ignore
-    # Changing 10-digits timestamp to 13-digits by padding with zeroes, since API supports 13-digits
-    return int(first_fetch_dt.timestamp()) * 1000
+    fetch_start_time_datetime = parse(fetch_start_time)  # type: ignore
+    start_fetch_timestamp = fetch_start_time_datetime.timestamp()  # type: ignore
+    if fetch_start_time_datetime.microsecond == 0:  # type: ignore
+        return int(start_fetch_timestamp) * 1000
+
+    timestamp = str(start_fetch_timestamp).replace('.', '')
+    if len(timestamp) == 11:
+        return int(timestamp) * 100
+    elif len(timestamp) == 12:
+        return int(timestamp) * 10
+    elif len(timestamp) == 13:
+        return int(timestamp)
+
+
+def timestamp_to_datetime_string(timestamp: int, include_miliseconds: bool = True):
```

Comment:
Can we use `timestamp_to_datestring` from csp instead?

Reply:
yes, good idea

Reply:
done